tensorflow/tools/ci_build/osx/arm64/tensorflow_as_test_release.Jenkinsfile

```diff
@@ -0,0 +1,167 @@
+/*
+Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+pipeline {
+    agent none
+    stages {
+        stage("Build Tensorflow") {
+            parallel {
+                stage("Python 3.8") {
+                    agent {
+                        label "silicon-ci"
+                    }
+                    environment {
+                        PYENV_ROOT="$HOME/.pyenv"
+                        PATH="$PYENV_ROOT/shims:/opt/homebrew/bin/:$PATH"
+
+                    }
+                    steps {
+
+                        sh '''
+                            echo 3.8.13 > /Users/admin/.python-version
+                            pyenv init -
+                            pyenv global 3.8.13
+                        '''
+
+                        sh 'python --version'
+
+                        git branch: "r2.13",
+                            url: "https://github.com/tensorflow/tensorflow.git"
+                            
+                        sh '''
+                            pip install --upgrade pip
+                            pip install -r ./tensorflow/tools/ci_build/release/requirements_mac.txt
+                        '''
+
+                        sh '''
+                        
+                            bazel --bazelrc="${WORKSPACE}/tensorflow/tools/ci_build/osx/arm64/.macos.bazelrc" test \
+                            --action_env PYTHON_LIB_PATH="/Users/admin/.pyenv/versions/3.8.13/lib/python3.8/site-packages" \
+                            --action_env PYTHON_BIN_PATH="/Users/admin/.pyenv/versions/3.8.13/bin/python3.8" \
+                            --config=macos_arm64 \
```

Comment:
The `--config=macos_arm64` lines can also be removed from this file since they are already present in "macos.bazelrc".